Type.sbvr

```diff
@@ -14,6 +14,9 @@ Term:       Big Integer
 Term:       Serial
 	Concept Type: Integer
 	Note: An auto-incrementing 'Integer'.
+Term:       Big Serial
+	Concept Type: Integer
```

Comment:
@joshbwlng I realized this right as I clicked approve that it should probably be
```
Concept Type: Big Integer
```
for clarity (although it'll work the same either way)